EscelForce.Business.Login/Interfaces/IUpdateMapService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using ExcelForce.Foundation.CoreServices.Models;
+using ExcelForce.Foundation.EntityManagement.Models.SfEntities;
+using System.Collections.Generic;
+
+namespace ExcelForce.Business.Interfaces
+{
+    public interface IUpdateMapService
```

Comment:
This service isn't needed. Please move all reusable functionalities across all 3 extract map implementations to `IExtractMapService`.